<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pokaz</class>
 <widget class="QDialog" name="Pokaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>618</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QComboBox" name="comboBox_pokaz"/>
   <widget class="QPushButton" name="pushButton_pokaz">
    <property name="text">
     <string>Pokaż</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QCheckBox" name="checkBox_pokaz">
    <property name="text">
     <string>Szukaj po nazwisku</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pokaz"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>100</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_zmien">
      <property name="text">
       <string>Zmień</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_usun">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
